google/cloud/spanner_v1/database.py

```diff
@@ -888,7 +898,13 @@ def run_in_transaction(self, func, *args, **kw):
         # done, flip the sanity check bit back.
         try:
             with SessionCheckout(self._pool) as session:
-                return session.run_in_transaction(func, *args, **kw)
+                observability_options = getattr(self, "observability_options", None)
+                with trace_call(
```

Comment:
Same as above comment.
This parent span should be above SessionCheckout to capture session related events